src/components/DataTable/DataTable.tsx

```diff
@@ -29,7 +29,7 @@ export interface DataTableProps {
   /** List of data types, which determines content alignment for each column. Data types are "text," which aligns left, or "numeric," which aligns right. */
   columnContentTypes: ColumnContentType[];
   /** List of column headings. */
-  headings: string[];
+  headings: (string | React.ReactNode)[];
```

Comment:
`React.ReactNode` already contains the type of `string`, so just `React.ReactNode[]` is sufficient